htdocs/expedition/card.php

```diff
@@ -648,7 +648,7 @@
 						$qty = "qtyl".$detail_batch->fk_expeditiondet.'_'.$detail_batch->id;
 						$batch_id = GETPOST($batch, 'int');
 						$batch_qty = GETPOST($qty, 'int');
-						if (!empty($batch_id) && ($batch_id != $detail_batch->fk_origin_stock || $batch_qty != $detail_batch->qty)) {
+						if (!empty($batch_id) && ($batch_id != $detail_batch->fk_origin_stock || $batch_qty > 0)) {
```

Comment:
What if you want to change qty in line of shipment detail from 5 to 0 ?

Reply:
You right, maybe should remove checks ? It's what i have done now